modules/social_features/social_group/social_group.module

```diff
@@ -211,6 +225,16 @@ function _social_group_get_group_labels() {
 
 }
 
+/**
+ * Implements hook_preprocess().
+ */
+function social_group_preprocess(array &$variables, string $hook): void {
+  /** @var \Drupal\social_group\JoinManagerInterface $manager */
+  $manager = \Drupal::service('plugin.manager.join');
```

Comment:
Should we add more context to this preprocess? Is this needed on all routes/paths? Is there any way to narrow this check?

Reply:
Done.